modules/drivers/oracle/test/metabase/driver/oracle_test.clj

```diff
@@ -231,3 +233,13 @@
                                                   &test_data_categories__via__cat.categories.id]
                                    :fk-field-id  (mt/id :venues :category_id)
                                    :fields       :none}]}))))))))
+
+(deftest oracle-connect-with-ssl-test
+  (mt/test-driver :oracle
+    (if (System/getenv "MB_ORACLE_SSL_TEST_SSL")
+      (testing "Oracle with SSL connectivity"
+        (tu/with-env-keys-renamed-by #(str/replace-first % "mb-oracle-ssl-test" "mb-oracle-test")
+          (metabase.query-processor-test.order-by-test/order-by-aggregate-fields-test)))
+      (println (u/colorize 'yellow (format "Skipping %s because %s env var is not set\n"
```

Comment:
there's a `u/format-color` function you can use that combines these

Reply:
Fixed in both tests.